tokenserver/assignment/sqlnode/sql.py

```diff
@@ -554,13 +544,16 @@ def add_node(self, service, node, capacity, **kwds):
             available = math.ceil(capacity * self.capacity_release_rate)
         res = self._safe_execute(sqltext(
             """
-            insert into nodes (service, node, available, capacity,
+            insert into nodes (id, service, node, available, capacity,
                             current_load, downed, backoff)
-            values (:service, :node, :available, :capacity,
+            values (:nodeid, :service, :node, :available, :capacity,
                     :current_load, :downed, :backoff)
             """),
-            service=service, node=node,
-            capacity=capacity, available=available,
+            nodeid=kwds.get('nodeid', None),
```

Comment:
This is to support the tests, by inserting the spanner node with a known `nodeid`.

Reply:
future nit: We should be consistent about `kw` `kwds` etc.